fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -6691,6 +6726,19 @@ public void truncateTable(TruncateTableStmt truncateTableStmt) throws DdlExcepti
         // tabletIdSet to save all newly created tablet ids.
         Set<Long> tabletIdSet = Sets.newHashSet();
         try {
+            // check replica quota if this operation done
+            long totalReplicaNum = 0;
```

Comment:
For `truncate` operation. I don't think we should check replica quota. Otherwise user may not be able to truncate table.